circle.yml

```diff
@@ -20,3 +20,7 @@ test:
     override:
         - cd $GS_WD && go test -v $(glide novendor)
         - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+        - cd $GS_WD &&
+          cp bindata.go bindata-repo.go &&
+          make update-bindata &&
+          cmp bindata.go bindata-repo.go
```

Comment:
What's more, if you are modifying the code of Kubernetes (or their related projects), you need to generate protobufs - for the same reason, they should be vendorable. They didn't figure any better way as well, you need to generate everything each time.